pkg/volume/testing.go

```diff
@@ -227,3 +231,22 @@ func (fd *FakeDeleter) Delete() error {
 func (fd *FakeDeleter) GetPath() string {
 	return fd.path
 }
+
+// FindEmptyDirectoryUsageOnTmpfs finds the expected usage of an empty directory existing on
+// a tmpfs filesystem on this system.
+func FindEmptyDirectoryUsageOnTmpfs() (*resource.Quantity, error) {
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "metrics_du_test")
```

Comment:
Yeah, I'll rename the method to be what it represents.  The issue isn't with any filesystem, really, it's with the assumptions baked into the test.